chapter_philosophy_and_refactoring.asciidoc

```diff
@@ -489,6 +504,11 @@ INSTALLED_APPS = [
 ----
 ====
 
+ // DAVID: This could just be "lists", which is more consistent with the other apps.
+ // See https://docs.djangoproject.com/en/4.2/ref/applications/#configuring-applications
+ //
+ // > by default, if Django finds exactly one AppConfig subclass in the apps.py submodule,
+ // > it uses that configuration for the application.
```

Comment:
yes i would prefer it.  i actually changed it because the new django tutorial told me to, iirc.